rds-aurora-mysql/README.md

```diff
@@ -0,0 +1,27 @@
+### Follow these steps to get started with federated SQL query against AWS RDS Aurora (MySQL Compatible).
```

Comment:
First line should be title. E.g. `# <title>`. Follow these steps should probably just be paragraph.

Reply:
updated